packages/shared/src/components/modals/SquadMemberModal.tsx

```diff
@@ -91,6 +101,29 @@ export function SquadMemberModal({
             </SimpleTooltip>
           );
         }}
+        initialItem={
```

Comment:
non-breaking suggestion:
I would suggest defining the item outside as a const and only passing that, think it will keep the overall component better structured.

Reply:
@rebelchris I was wondering on which would be prefferd. I am fine with top level but maybe moving it to separate component is also good since many components in the same file also look messy. I guess it comes down to once it needs to be resed in multiple things we then move it outside.

Reply:
I think it will be something we need to document as it's "vague" at best now.

Normally how I look at it:
- small component can be inline
- large components (30+ lines?) can be own components
- reusable small components can be own components.

So think for now inside this components function is fine, just outside of the render for now.
Buit love to hear your thoughts on this as well.

Reply:
Yeah, let's hard document this. it would be better to linter it up, but option though.